news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges.md

```diff
@@ -0,0 +1,83 @@
+
+### Contest Judges - Players
+
+Attention to all professional osu! players! A brand new section called **Players' Pick** is making its appearance for the osu! Beatmapping World Championship.
+
+Players' Pick is a new side-competition that will be organized in parallel to the primary contest. Aside from the conventional contest winners, we will consult a handful of professional circle clickers on evaluating each submission as of the second round of the contest, eventually determining a winning team that will receive the Players' Pick award!
+
+"How?" you say? Well, obviously, our circle-clicking specialists will be evaluating these submissions under a set of criteria, which are listed below:
+
+- **Sophistication**: This criterion will focus on how comfortable it is to play the beatmap, according to its difficulty level. As much as they can be considered pieces of art, beatmaps are defined as in-game levels made for users to enjoy them. Therefore, for them to gain players' attraction, they need to be player-friendly first.
+- **Profundity**: Players are supposed to rate the quality of the beatmap by looking at nothing but its structure. Playing a super-simple jump/stream map might be fun, but we are here to have a competition here after all! Establishing complex mapping structures can take vast amounts of time and effort. Thus players are supposed to take the profundities of the beatmaps into account during their evaluation.
+- **Uniqueness**: Throughout the history of osu!, we can see that beatmaps displaying unique patterns usually be the centre of community attention and sometimes even become era-defining masterpieces. With that in mind, we would like to draw players' attention to the artistic aspects of mapping and see where their opinions revolve.
+
+Additionally, we will also entertain a **Player's Impression** section. Players are asked to note down their personal thoughts and subjective opinions regarding the beatmap and rate it accordingly.
+
+To take part as a Players' Pick judge, your osu! rank needs to be *higher than #5000* as the initial requirement. We're planning to recruit **5 judges** in total for this section of the contest. So if you're a professional osu! player and like playing a variety of beatmaps, don't miss out on becoming a Players' Pick judge!
+
+### Multi-mode Judges
+
+Drummers, fruit catchers, and keyboard smashers behold! This year, we are going to host three more contests for three other game modes. The details are still worked on, but if you're an early bird wishing to partake as a judge for osu!taiko, osu!catch, or osu!mania edition of the osu! Beatmapping World Championship, you may submit your application right away!
+
+## Schedule
+
+Below is the provisional schedule for the upcoming contest. Be reminded that reschedules might occur due to unexpected organizational matters.
```

Comment:
```suggestion
Below is the provisional schedule for the upcoming contest. Keep in-mind that reschedules might occur due to unexpected organizational matters.
```
nit

Reply:
"keep in mind"? why hyphenate?

Reply:
afaik "in-mind" and "in mind" are both correct; just down to preference of style (like video game vs videogame vs video-game); though could be totally wrong